### PR TITLE
Add a new Timestamp Format option

### LINKED ISSUE #25: Support custom timestamp attachment format
Currently, the timestamp attachment is not formatted and is just written as `DateTimeOffset.ToString()`, which is depending on the culture of the current thread.

https://github.com/mgibas/serilog-sinks-slack/blob/master/Serilog.Sinks.Slack/SlackSink.cs#L89
```cs
new Field{Title = "Timestamp", Value = logEvent.Timestamp.ToString(), Short = _options.DefaultAttachmentsShortFormat}
```

Would it be possible to add support for custom timestamp attachment formats, so you can use any format you like, without having to rely on the current machine culture?

### PR DESCRIPTION
**Pull request by kimsey0**
Fixes #25

### CODE REVIEW

**Comment by kimsey0** [comment]
@TrapperHell, how does this look? 🙂

File: Serilog.Sinks.Slack/SlackSink.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ protected IEnumerable<Attachment> CreateAttachments(LogEvent logEvent)
                     Fields = new List<Field>()
                 };
 
-                AddAttachmentField(ref attachment, new Field { Title = "Level", Value = logEvent.Level.ToString(), Short = _options.DefaultAttachmentsShortFormat });
+                AddAttachmentField(ref attachment, new Field { Title = "Level", Value = logEvent.Level.ToString(_options.TimestampFormat), Short = _options.DefaultAttachmentsShortFormat });
                 AddAttachmentField(ref attachment, new Field { Title = "Timestamp", Value = logEvent.Timestamp.ToString(), Short = _options.DefaultAttachmentsShortFormat });
```
Comment (TrapperHell) [inline Serilog.Sinks.Slack/SlackSink.cs:93]: I believe that you've added the format to the wrong field. This was intended for the `Timestamp` field, not `Level`, right?

**Review by TrapperHell — CHANGES_REQUESTED** [review]

File: Serilog.Sinks.Slack/SlackSink.cs (hunk 1 above)
Comment (kimsey0) [reply Serilog.Sinks.Slack/SlackSink.cs]: You are correct. Whoops! Sorry about that. Fixed now.

**Comment by TrapperHell** [comment]
This is looking good now @kimsey0 . Has it perchance been tested at all, since I currently cannot test this?

**Comment by kimsey0** [comment]
I have now, and it seems to work as expected. Here are posted messages with the roundtrip (`"o"`), short (`d`) and default (`null`) date format, respectively:
Slack messages with different date formats
(All from my Danish-locale computer.)

**Review by TrapperHell — APPROVED** [review]

**Comment by TrapperHell** [comment]
Looks good to me. Thank you for your contribution! Just one last thing - if I may - bump up the .csproj version to 2.0.2 so that we can have your build out on nuget.

**Comment by kimsey0** [comment]
For sure - done!

Thank you for taking over maintenance on this project and for handling my pull request so quickly. 😄

**Comment by mgibas** [comment]
@kimsey0 I feel attacked! jk - also happy that this project is alive 😄 🎉

**Comment by kimsey0** [comment]
@mgibas: Sorry, I must have misunderstood this https://github.com/mgibas/serilog-sinks-slack/pull/24#issuecomment-753907188. 😜

**Pull request merged**